src/lib/notifications.js

```diff
@@ -0,0 +1,68 @@
+import { dispatchCustomEvent } from './events.js';
+import { i18n } from './i18n.js';
+
+/**
+ * @typedef {import('../toast/types.js').Toast} Toast
+ */
+
+/**
+ * @param {Toast} detail
+ * @param {Window|Document|HtmlElement} target
+ */
+export function notify (detail, target = window) {
```

Comment:
I'm wondering if this whole file should and methods should be named `notify`. What about `emitToast` or `dispatchToast`?